react/react1/week1/my-app/src/Listitems.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+export function Listitems(props){
+    return (
```

Comment:
ListItem singular and not ListItem**s**, right?

Reply:
hi thank you for the feedback. I have now made this as Listitem singular as suggested.